code/modules/hydroponics/grown/mushrooms.dm

```diff
@@ -197,6 +197,21 @@
 	desc = "<I>Cantharellus Cibarius</I>: These jolly yellow little shrooms sure look tasty!"
 	icon_state = "chanterelle"
 
+/obj/item/food/grown/mushroom/chanterelle/attackby(obj/item/I, mob/user, params)
+	if(!istype(I, /obj/item/kitchen/spoon))
+		return ..()
+
+	if(seed.potency > 95)
+		to_chat(user, span_notice("You hollow up the chanterelle with [I]."))
+		remove_item_from_storage(user)
+		qdel(src)
+		if(seed.resistance_flags & FIRE_PROOF)
+			user.put_in_hands(/obj/item/clothing/head/wizard/chanterelle/fr)
+		else
+			user.put_in_hands(/obj/item/clothing/head/wizard/chanterelle)
+
+
```

Comment:
you should early return on the potency check, also you should remove some of the trailing newlines because I am nitpickey